src/Mapbender/CoreBundle/Element/Type/FeatureInfoAdminType.php

```diff
@@ -14,6 +14,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('displayType', 'Symfony\Component\Form\Extension\Core\Type\ChoiceType', array(
                 'required' => true,
+                'label' => 'mb.core.featureinfo.admin.displaytype',
                 'choices' => array(
                     'Tabs' => 'tabs',
```

Comment:
Translationf for Tabs and Accordion is missing

Reply:
Fixed :)